operations/iuf/workflows/management_rollout.md

```diff
@@ -426,7 +473,7 @@ The worker canary node can be any worker node and does not have to be `ncn-w001`
 1. (`ncn-m001#`) Use `kubectl` to remove the `iuf-prevent-rollout=true` label from the canary node.
 
     ```bash
-    kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout-
+    kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout
```

Comment:
The `-` at the end of the line is required to remove the label.   This command should be what it was before:  ```
kubectl label nodes "${WORKER_CANARY}" --overwrite iuf-prevent-rollout-
```

Reply:
Modified required changes